python/foglamp/services/south/server.py

```diff
@@ -110,6 +111,16 @@ def __init__(self):
 
             # Plugin initialization
             self._plugin_info = self._plugin.plugin_info()
+            if self._plugin_info['version'].startswith('2') and self._plugin_info['mode'] == 'async':
+                _LOGGER.warning('Plugin %s should be loaded by C version of South server', self._name)
+                import foglamp.services.south.modify_process as mp
+                await mp.disable_service(self._name)
+                await mp.unregister_service(self._name, self._core_management_port, self._microservice_id)
+                await asyncio.sleep(3)
```

Comment:
sleep or poll to wait for getting it unregistered  (pass on ServiceDoesNotExist in serviceRegistry)?